.travis.yml

```
@@ -39,3 +39,9 @@ jobs:
         - ./build.sh pull squid-ssl
         - ./build.sh build squid-ssl
         - ./build.sh push squid-ssl
+    - stage: deploy
+        name: jdk11-datadog-agent
```

It could be too much preciousness, but more specific image tags are more descriptive and can helps on flexibility on the future.

```suggestion
        name: 11.0.7-jre-slim-buster-datadog-agent
```

---

:+1: